src/crypto/tls/common.go

```diff
@@ -125,8 +126,18 @@ const (
 	CurveP384 CurveID = 24
 	CurveP521 CurveID = 25
 	X25519    CurveID = 29
+	SIKEp434  CurveID = CurveID(kem.SIKEp434)
+	Kyber512  CurveID = CurveID(kem.Kyber512)
```

Comment:
I disagree on this one. These are the kem algorithms that can used in combination, for example, with pq signatures (which will be another experiment we will run), in which kem are used for confidentiality. So they are not "kemtls" per se ;)

Reply:
True, but they're also not actually "curves". KEMTLS is already overloading the semantics of the `CurveID` by adding codepoints for KEMs. To make this fact clear, I would at least add the prefix "KEM" to the variables. I think "KEMTLS" is better because it makes it clear that these code points are used to negotiate KEMTLS and not TLS.

Reply:
Another advantage is that it's easier to search through the code for variables related to KEMTLS.